protocol/x/affiliates/keeper/keeper_test.go

```diff
@@ -14,3 +20,193 @@ func TestLogger(t *testing.T) {
 	logger := tApp.App.AffiliatesKeeper.Logger(ctx)
 	require.NotNil(t, logger)
 }
+
+func TestRegisterAffiliate(t *testing.T) {
```

Comment:
Fine if we are testing both `RegisterAffiliate` and `GetReferredBy` here; modify function name to be explicit (`TestRegisterAffiliate_GetReferredBy`)